infoschema/tables.go

```diff
@@ -680,6 +685,15 @@ var tableTiDBClusterConfigCols = []columnInfo{
 	{"VALUE", mysql.TypeVarchar, 128, 0, nil, nil},
 }
 
+var tableTiDBClusterLoadInfoCols = []columnInfo{
+	{"TYPE", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"ADDRESS", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"DEVICE", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
Is better to change the `DEVICE` to `DEVICE_TYPE` and `NAME` to `DEVICE_NAME`? @bb7133 @djshow832